packages/ui/brand/src/components/ComposerLogo/ComposerLogo.stories.tsx

```diff
@@ -12,25 +13,35 @@ import { Button } from '@dxos/react-ui';
 import { mx } from '@dxos/react-ui-theme';
 import { withTheme } from '@dxos/storybook-utils';
 
-import { ComposerLogo, type AnimationController } from './ComposerLogo';
-// https://pixabay.com/sound-effects/search/logo/?pagi=2
-// @ts-ignore
-import ident from '../../../assets/sounds/ident-1.mp3';
+import { type AnimationController, ComposerLogo, ComposerSpinner } from './ComposerLogo';
 import { DXOS } from '../../icons';
 
+// import ident from '../../../assets/sounds/ident-1.mp3';
+
+// https://pixabay.com/sound-effects/search/logo/?pagi=2
+
 export default {
   title: 'brand/Logo',
   component: ComposerLogo,
   decorators: [withTheme],
 };
 
+// TODO(burdon): Get from theme?
+const colors = {
+  gray: '#888888',
+  purple: '#AA23D3',
+  orange: '#CA6346',
+  green: '#4DA676',
+  blue: '#539ACD',
+};
```

Comment:
It’s okay to use absolute colors in stories IMO, though it is (or should be) trivial to get them from `react-ui-theme`.

Reply:
fwiw this relates to my (and Daniel’s) idea to replace colors even in Tailwind with CSS variables, which would eliminate the need to implement semantic tokens in Tailwind and make it easier to reference them from TS. I have a system (vite plugin + package) ready to go for this.